app/controllers/analytics/districts_controller.rb

```diff
@@ -2,7 +2,11 @@ class Analytics::DistrictsController < AnalyticsController
   before_action :set_organization_district
 
   def show
-    @analytics = Rails.cache.fetch(analytics_cache_key) { analytics }
+    if ENV.fetch('SIMPLE_SERVER_ENV') == 'qa'
```

Comment:
```suggestion
    if FeatureToggle.enabled(:uncached_queries_for_dasboard)
```

Reply:
@govindkrjoshi Added new env variable `ENABLE_CACHED_QUERIES_FOR_DASHBOARD`. Toggling on the same.